# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/crashtests/crashtests.list
# Commit: b32811ea1c61
# Full Hash: b32811ea1c61e1891454d826f7ba003416c9eb57
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-03-11 21:13:39
# Regressor Bug: 1860328
# File Overlap Count: 2
# Description:
#   Bug 1860328 - Fix a crash which a caret frame misses a MarkNeedsDisplayItemRebuild() call r=emilio
#   
#   Generally, we mark a caret frame for display first, and then
#   nsCaret tracks this frame in nsCaret::SchedulePaint to call
#   MarkNeedsDisplayItemRebuild() accordingly. However, it's possible
# ==============================================================================

diff -r caab023e5d2d -r b32811ea1c61 layout/generic/crashtests/crashtests.list
--- a/layout/generic/crashtests/crashtests.list	Mon Mar 11 13:41:27 2024 +0000
+++ b/layout/generic/crashtests/crashtests.list	Mon Mar 11 13:41:28 2024 +0000
@@ -812,3 +812,4 @@
 load 1821603.html
 load 1822118.html
 load 1825434.html
+load 1878805.html